src/contract/contract.ts

```diff
@@ -30,10 +30,12 @@ interface ValidatorAppSchema {
   numGlobalByteSlices: any;
 }
 
-export enum ContractVersion {
-  V1_1 = "v1_1",
-  V2 = "v2"
-}
+export type ContractVersion = "v1_1" | "v2";
+
+export const CONTRACT_VERSION: Record<string, ContractVersion> = {
+  V1_1: "v1_1",
+  V2: "v2"
+};
```

Comment:
```suggestion
export const CONTRACT_VERSION = {
  V1_1: "v1_1",
  V2: "v2"
} as const;
export type ContractVersionValue = ValueOf<typeof CONTRACT_VERSION>;
```
@Anlerkan I think ideally we should only have these. 
In this version we also have type checking while using `CONTRACT_VERSION.V1_1` since it's a `const` object.

But does this cause problems while exporting types? I realized we define some types in both files and `.d.ts` files. This didn't feel very safe, do you know if/why that is necessary ?

Reply:
Thanks for the review @jamcry! I updated the constant and the type. I don't know if it is necessary or not.

Reply:
I think I was actually checking the dist files 🤦‍♂ So no issues